src/spaces/gradedspace.jl

```diff
@@ -149,6 +149,12 @@ function ⊕(V₁::GradedSpace{I}, V₂::GradedSpace{I}) where {I<:Sector}
     return typeof(V₁)(dims; dual=dual1)
 end
 
+function ⊖(V::GradedSpace{I}, W::GradedSpace{I}) where {I<:Sector}
+    V ≿ W && isdual(V) == isdual(W) ||
+        throw(SpaceMismatch("$(W) is not a subspace of $(V)"))
+    return typeof(V)(c => dim(V, c) - dim(W, c) for c in sectors(V))
```

Comment:
The `isdual` flag of `V` (and `W`, as it is required to be the same) is lost in this construction. See the `⊕` implementation.